Formula/cartridge-cli.rb

```diff
@@ -0,0 +1,20 @@
+class CartridgeCli < Formula
+  desc "Tarantool Cartridge command-line utility"
+  homepage "https://tarantool.org/"
+  url "https://github.com/tarantool/cartridge-cli/archive/1.2.0.tar.gz"
+  sha256 "4668f84a779eb8f6ddfcafaf4c727899ef2fe31622e8999317c512b685bd1c31"
+  head "https://github.com/tarantool/cartridge-cli.git"
+
+  depends_on "cmake" => :build
+  depends_on "tarantool"
+
+  def install
+    system "cmake", ".", *std_cmake_args, "-DVERSION=#{version}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"cartridge", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Hi, and thank you for the feedback!

All functions except `--help` and `--version` create files. Is it ok to match output of `--help` against some pattern?

Reply:
No, it is acceptable to see if a file is created and matches a pattern though.

Reply:
Updated. Please take a look.